src/support/timefuncs.c

```diff
@@ -28,31 +28,32 @@
 extern "C" {
 #endif
 
-#if defined(_OS_WINDOWS_)
-static double floattime(void)
+int jl_gettimeofday(struct jl_timeval *jtv)
```

Comment:
`clock_now` is also used in `src/gc.c`. Does it seem reasonable to update those calls too?